Add A Labour of Love contest news post (#4530)
ppy/osu-wiki: news/2020-11-29-a-labour-of-love.md

---
layout: post
title: "A Labour of Love"
date: 2020-11-29 08:00:00 +0000
---

Are you a mapper? A storyboarder? Or maybe a talented music producer? Show your love for an osu! community favorite in this multi-layered contest!

![](/wiki/shared/news/banners/alol_cover.jpg)

Everyone knows [**Kuba Oms - My Love**](https://osu.ppy.sh/beatmapsets/163112). Put 100 osu! players in a room and you'd be safe betting that half of them could sing its chorus. The beatmap has nearly 70 *million* plays and is a source of nostalgia for generations of us.

We paired with *Kuba Oms* to set up a contest celebrating this timeless classic — a contest more ambitious than any we've hosted before!

## Teams

Lots to go through. Let's keep it short.

**This is a team-based contest.** Users with different skillsets must collaborate to create a *full beatmap experience*, including:

- Remix of the song **Kuba Oms - My Love** (stems provided)

> - Single-difficulty beatmap (osu! only)

Reviewer: Are you sure we want to limit it to one difficulty? Is this for ease of judging?

> - Storyboard or custom video

Author: cant find a better place to put this near the top so gonna leave it

> **Submit your team name and member list to [this form](https://docs.google.com/forms/d/e/1FAIpQLScjtVsUWIArvf--pKA0RLiCdXuO1_wO2Va2ICjenEcz9ZqI5Q/viewform).** Once confirmed, you will receive the the stems for *My Love* and your team will be documented on the [A Labour of Love contest wiki page](/wiki/Contests/A_Labour_of_Love_Contest).

Reviewer 2: may want to provide an alternative way to submit (contacting u?) cuz china + google

With teams being public, we'll be taking a different approach from usual contests: **entries will NOT be anonymized.**

> In fact, we encourage you to tease your submission as it's being created! Feel free to announce your team publicly and show off your work. We're celebrating an osu! classic and we can't very well do that if everyone keeps their work hidden.

Reviewer 2: ```suggestion
In fact, we encourage you to tease your submission as it's being created! Feel free to announce your team publicly and show off your work. We're celebrating an osu! classic and we can't do that very well if everyone keeps their work hidden.
```

*"I'm a super cool mapper/storyboarder, but I don't know anyone who can remix songs. What do I do?"* There's likely a lot of people in this situation. We recommend anyone looking for teammates to post in the comments saying so. Forming a suitable team is half of the challenge for this contest!

## Schedule

> - **3 months:** Contest is open for submissions [here](https://osu.ppy.sh/community/contests/115) until 2021-02-28. There's a lot of work going into this, so you might need the extra time.

Reviewer 2: ```suggestion
- **3 months:** Contest is open for submissions [here](https://osu.ppy.sh/community/contests/115) until the end of February. There's a lot of work going into this, so you might need the extra time.
```

- **2 weeks:** Submissions are judged by separate mapper and musician judging panels.

> - **2 weeks:** Community vote is opened for top 5 overall submissions

Reviewer 2: ```suggestion
- **2 weeks:** Community voting is opened for top 5 overall submissions.
```

> Once each of these are completed, the winners will be announced!

Reviewer 2: ```suggestion
Once all of these are completed, the winners will be announced!
```

## Prizes

> No such thing as a contest without prizes! Unlike most contests however, there's a few different way to earn prizes this time around.

Reviewer 2: ```suggestion
No such thing as a contest without prizes! Unlike most contests however, there's a few different ways to earn prizes this time around.
```

> 2 groups will handle judging: **mappers (including storyboarders!) and musicians.** They'll be expected to evaluate the full package — the remixed song + beatmap + storyboard/video combo. 

Reviewer 2: ```suggestion
Two groups will handle judging: **mappers (including storyboarders!)** and **musicians**. They'll be expected to evaluate the full package — the remixed song + beatmap + storyboard/video combo. 
```

Each group has its own specialty of course, so we'll be rewarding prizes to the all members of the highest scoring team from each panel.

- **Mapper panel**
    - **Runner-up:** 3 months of osu!supporter 
    - **Winner:**  6 months of osu!supporter + *Mappers' Pick* profile badge

> ![Mappers' Pick profile badge](/wiki/shared/news/2020-11-29-a-labour-of-love/alol_mapperspick@2x.png)

Reviewer 2: ```suggestion
![Mappers' Pick profile badge](/wiki/shared/news/2020-11-29-a-labour-of-love/alol_mapperspick.png)
```
this is handled by the website

- **Musician panel**
    - **Runner-up:** 3 months of osu!supporter 
    - **Winner:**  6 months of osu!supporter + *Musicians' Pick* profile badge

> ![Musicians' Pick profile badge](/wiki/shared/news/2020-11-29-a-labour-of-love/alol_musicianspick@2x.png)

Reviewer 2: ```suggestion
![Musicians' Pick profile badge](/wiki/shared/news/2020-11-29-a-labour-of-love/alol_musicianspick.png)
```

> The legend himself, *Kuba Oms*, will be part of this panel!

Reviewer 2: ```suggestion
**The legend himself, *Kuba Oms*, will be part of this panel!**
```
seems epic to me

That's not all though! When panel results are in, we'll combine them to form the overall top 5 entries. These will be open for a *community vote*, meaning you can pick your favorite entry!

Once the vote is finalized, we'll combine panel scores with community vote scores to determine the overall winner (weighting for each category is undecided).

This winner will be announced during a **live stream showcase of the finalists** with commentary from some of the judges!

- **Overall prizes**
    - **Runner-up:** 3 months of osu!supporter + *Overall runner-up* profile badge
    - **Winner:**
        - 3 months of osu!supporter
        - *Overall winner* profile badge

>         - *Elite Mapper*, *Elite Storyboarder*, and *Elite Remixer* user titles

Reviewer 2: "osu!mixer" was previously used for a musician title. if "Elite Remixer" isn't specifically important i recommend using "osu!mixer" again

>         - Submission fast-tracked to Loved status (or Ranked if applicable!)

Reviewer: Yeah, was also going to mention this one. I don't think we should be promoting aiming for loved status for a contest like this.

> ![Overall runner-up profile badge](/wiki/shared/news/2020-11-29-a-labour-of-love/alol_overallrunnerup@2x.png) ![Overall winner profile badge](/wiki/shared/news/2020-11-29-a-labour-of-love/alol_overallwinner@2x.png)

Reviewer 2: ```suggestion
![Overall runner-up profile badge](/wiki/shared/news/2020-11-29-a-labour-of-love/alol_overallrunnerup.png) ![Overall winner profile badge](/wiki/shared/news/2020-11-29-a-labour-of-love/alol_overallwinner.png)
```

> Also in case one team wins both a specific panel and overall, the overall prizes will take priority!

Reviewer 2: ```suggestion
Also, in case one team wins both a specific panel and overall, the overall prizes will take priority!
```

## Rules

> Follow these rules if you don't want your hard work to be disqualified:

Reviewer 2: considering the whole process is public from the beginning and that it's a pretty big effort compared to other contests, I think you should notify teams if their submission would be disqualified (provided they submit early, I guess) so they have time to fix it

- **2 weeks:** Submissions are judged by separate mapper and musician judging panels.
- **2 weeks:** Community voting is opened for top 5 overall submissions.

Once all of these are completed, the winners will be announced!

## Prizes

No such thing as a contest without prizes! Unlike most contests however, there's a few different ways to earn prizes this time around.

Two groups will handle judging: **mappers (including storyboarders!)** and **musicians**. They'll be expected to evaluate the full package — the remixed song + beatmap + storyboard/video combo. 

Each group has its own specialty of course, so we'll be rewarding prizes to the all members of the highest scoring team from each panel.

- **Mapper panel**
    - **Runner-up:** 3 months of osu!supporter 
    - **Winner:**  6 months of osu!supporter + *Mappers' Pick* profile badge

![Mappers' Pick profile badge](/wiki/shared/news/2020-11-29-a-labour-of-love/alol_mapperspick.png)

- **Musician panel**
    - **Runner-up:** 3 months of osu!supporter 

>     - **Winner:**  6 months of osu!supporter + *Musicians' Pick* profile badge

Reviewer 3: shouldnt the runner-up prize be below the winner?

Author: i think this layout works better since badge images are in the next line

![Musicians' Pick profile badge](/wiki/shared/news/2020-11-29-a-labour-of-love/alol_musicianspick.png)

> **The legend himself, *Kuba Oms*, will be part of this panel!**

Reviewer 3: ```suggestion
**The legend himself, *Kuba Oms*, will be a part of the Musician panel!**
```

This winner will be announced during a **live stream showcase of the finalists** with commentary from some of the judges!

- **Overall prizes**
    - **Runner-up:** 3 months of osu!supporter + *Overall runner-up* profile badge
    - **Winner:**
        - 6 months of osu!supporter
        - *Overall winner* profile badge
        - *Elite Mapper*, *Elite Storyboarder*, and *osu!mixer* user titles
        - Submission bundled with new osu! downloads and fast-tracked to Ranked status

![Overall runner-up profile badge](/wiki/shared/news/2020-11-29-a-labour-of-love/alol_overallrunnerup.png) ![Overall winner profile badge](/wiki/shared/news/2020-11-29-a-labour-of-love/alol_overallwinner.png)

Also, in case one team wins both a specific panel and overall, the overall prizes will take priority!

## Rules

Follow these rules if you don't want your hard work to be disqualified. We'll notify teams about potential disqualification early so they can fix any issues.

- **Submission must be a "full beatmap experience" including the following:**
    - Remix of the song **Kuba Oms - My Love**

>     - Beatmap spread following the [ranking criteria](/wiki/Ranking_Criteria).

Reviewer 3: ```suggestion
    - Beatmap spread following the [ranking criteria](/wiki/Ranking_Criteria)
```

Keep in mind that it is 100% acceptable to tease your submission publicly in this contest. You're making something cool, so you deserve to show it off!

> ## Remix usage rights

Reviewer: this should probably be on the wiki page too, under rules or its own heading
